README.md

```diff
@@ -33,15 +33,24 @@ fastify.listen(3000, err => {
 })
 ```
 
-In case a client reaches the maximum number of allowed requests, a standard Fastify error will be returned to the user with the status code setted to `429`:
+In case a client reaches the maximum number of allowed requests, an error will be send to the user with the status code setted to `429`:
```

Comment:
```suggestion
In case a client reaches the maximum number of allowed requests, an error will be sent to the user with the status code set to `429`:
```